building/trusted_applications.rst

```diff
@@ -338,6 +338,68 @@ sequence:
         }
     }
 
+
+Identifying TA's client
+***********************
+
+GP TEE specification is mades so that TEE sessions, once established, are
+reliable. A TA instance can identify its client login method when session is
+opened. Based on the login, the TA can accpet of not the session. A TA can
```

Comment:
s/accpet/accept/

But the sentence itself is probably wrong, I don't get what you intend to say, can you perhaps rephrase it?